examples/Bremsstrahlung/submit/0008gpus.cfg

```diff
@@ -0,0 +1,70 @@
+# Copyright 2013-2017 Heiko Burau, Richard Pausch, Felix Schmitt, Axel Huebl
+#
+# This file is part of PIConGPU.
+#
+# PIConGPU is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PIConGPU is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PIConGPU.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+
+
+##
+## This configuration file is used by PIConGPU's TBG tool to create a
+## batch script for PIConGPU runs. For a detailed description of PIConGPU
+## configuration files including all available variables, see
+##
+##                      doc/TBG_macros.cfg
+##
+
+
+#################################
+## Section: Required Variables ##
+#################################
+
+TBG_wallTime="8:00:00"
+
+TBG_gpu_x=8
+TBG_gpu_y=1
+TBG_gpu_z=1
+
+TBG_gridSize="-g 2048 2048 1"
+TBG_steps="-s 5000"
+
+TBG_periodic="--periodic 0 0 0"
+
+#################################
+## Section: Optional Variables ##
+#################################
+
+
+TBG_plugins="--hdf5.period 1000  \
+             --e_macroParticlesCount.period 1000 \
```

Comment:
Didn't you use energy histograms and the particle calorimeter plugin for analysis of the output?